### PR TITLE
feat: enable connection to database

### PR DESCRIPTION
**Pull request by SkellyBG**

### CODE REVIEW

File: src/db/schema.sql [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+SELECT pg_catalog.set_config('search_path', '', false);
+SET check_function_bodies = false;
+SET xmloption = content;
+SET client_min_messages = warning;
+SET row_security = off;
+
+
+CREATE TABLE IF NOT EXISTS public.teams (
+    team_name text PRIMARY KEY,
+    team_channel text NOT NULL,
```
Comment (eluric) [inline src/db/schema.sql:15]: Change to include the category channel ID, voice channel ID and text channel ID as well as the team role ID

**Review by eluric — CHANGES_REQUESTED** [review]

File: src/db/schema.sql [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    category_channel_id text NOT NULL,
+    voice_channel_id text NOT NULL,
+    text_channel_id text NOT NULL,
+    team_role_id text NOT NULL,
+    puzzle_solved integer NOT NULL default 0
+);
+
+CREATE TABLE public.players (
+    discord_id text PRIMARY KEY,
+    player_name text NOT NULL,
```
Comment (eluric) [inline src/db/schema.sql:28]: Remove player_name

Comment (eluric) [inline src/db/schema.sql]: Remove player_name

**Review by eluric — APPROVED** [review]

File: src/db/schema.sql (hunk 2 above)
Comment (SkellyBG) [reply src/db/schema.sql]: done!

**Pull request merged**